microservices/workers/src/Worker/Asterisk.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Worker;
+
+use Ivoz\Ast\Domain\Job\AriJobInterface;
+use Ivoz\Ast\Infrastructure\Asterisk\ARI\ARIConnector;
+use Ivoz\Core\Application\RegisterCommandTrait;
+use Ivoz\Core\Application\RequestId;
+use Ivoz\Core\Domain\Service\DomainEventPublisher;
+use Ivoz\Core\Infrastructure\Persistence\Redis\RedisMasterFactory;
+use Symfony\Component\HttpFoundation\Response;
+use Psr\Log\LoggerInterface;
+
+class Asterisk
+{
+    use RegisterCommandTrait;
+
+    public function __construct(
+        DomainEventPublisher $eventPublisher,
```

Comment:
private DomainEventPublisher $eventPublisher

Reply:
I would do this for all exitings workers in a future Pull Request, because the definition of the attributes is part of the trait displayed above and declaring them private here will force me to remove them from the trait.

This will also affect the other existing workers, so I prefer to handle all of them together.

Reply:
:+1: